doc/post-transaction-actions.rst

```diff
@@ -0,0 +1,90 @@
+..
+  Copyright (C) 2019 Red Hat, Inc.
+
+  This copyrighted material is made available to anyone wishing to use,
+  modify, copy, or redistribute it subject to the terms and conditions of
+  the GNU General Public License v.2, or (at your option) any later version.
+  This program is distributed in the hope that it will be useful, but WITHOUT
+  ANY WARRANTY expressed or implied, including the implied warranties of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+  Public License for more details.  You should have received a copy of the
+  GNU General Public License along with this program; if not, write to the
+  Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+  02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+  source code or documentation are not subject to the GNU General Public
+  License and may only be used or replicated with the express permission of
+  Red Hat, Inc.
+
+===================================
+DNF post-transaction-actions Plugin
+===================================
+
+-----------
+Description
+-----------
+
+Plugin runs actions (shell commands) after transaction is completed.
+Actions are defined in action files.
+
+.. note:: The same shell command (after variables substitution) is executed only once per transaction.
+   The order of execution of the commands may differ from the order of the packages in the transaction.
+
+-------------
+Configuration
+-------------
+
+``/etc/dnf/plugins/post-transaction-actions.conf``
+
+.. code-block:: ini
+
+   [main]
+   enabled = 1
+   actiondir = /etc/dnf/plugins/post-transaction-actions.d/
+
+The ``enabled`` is used to enable/disable plugin. If it is omitted default value "1" is used.
+The ``actiondir`` sets path to directory with action files. Action file must have extension ".action".
+If ``actiondir`` is omitted default value "/etc/dnf/plugins/post-transaction-actions.d/" is used.
+
+------------------
+Action file format
+------------------
+
+Empty lines and lines starts with '#' character are ignored.
+Each non-comment line in these files contains the following three fields: action_key, transaction_state,
+command. The fields are separated by ':' character.
+
+line format:
+  ``action_key:transaction_state:command``
```

Comment:
I'd leave the line format (the two above lines) out, I've merged that with the previous paragraph ...